content/tools/polyfill.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Polyfills
+sub: Cascade Layers, CSS Anchor Positioning, CSS Toggles, Popover
+feature: true
+date: 2023-10-31
+image:
+  position: top
+  src: projects/polyfill-demo.jpg
+oss: owner
+author: oddbird
+action:
+  text: Explore Polyfills »
+  url: https://github.com/orgs/oddbird/repositories?q=polyfill&type=all&language=&sort=
+index: Polyfill
+links:
+  source: https://github.com/orgs/oddbird/repositories?q=polyfill&type=all&language=&sort=
+summary: |
+  Along with our work for the W3C
+  developing specifications
+  for the CSS language,
+  OddBird has started and maintains
+  a number of powerful CSS polyfills
+  including Cascade Layers, CSS Anchor Positioning,
+  CSS Toggles, and Popover Attribute.
+---
+
+{% import 'embed.macros.njk' as embed %}
+
+## Cascade Layers Polyfill
+
+[Demo](https://www.npmjs.com/package/@csstools/postcss-cascade-layers) |
+[Source](https://github.com/oddbird/cascade-layers-polyfill-demo)
+
+PostCSS Cascade Layers lets you use @layer
+following the Cascade Layers Specification.
+For more information on layers,
+checkout [A Complete Guide to CSS Cascade Layers](https://www.oddbird.net/2022/02/21/layers-guide/)
+by Miriam Suzanne.
+
+## CSS Anchor Positioning Polyfill
+
+[Demo](https://anchor-polyfill.netlify.app/) |
+[Source](https://github.com/oddbird/css-anchor-positioning)
+
+The [CSS anchor positioning specification](https://www.w3.org/TR/css-anchor-position-1/)
+defines anchor positioning,
+"where a positioned element can size and position itself
+relative to one or more 'anchor elements'
+elsewhere on the page."
+This CSS Anchor Positioning Polyfill
+supports and is based on this specification.
+
+The proposed anchor() and anchor-size() functions
+add flexibility to how absolutely positioned elements
+can be placed within a layout.
+Instead of being sized and positioned
+based solely on the position of their containing block,
+the proposed new functions allow absolutely positioned elements
+to be placed relative to one or more
+author-defined anchor elements.
+
+## CSS Toggles
```

Comment:
I would move this to the end, or remove it entirely. I don't think it's deprecated for now - I think it's pretty well abandoned at the spec level. We can mention that we built a prototype for exploration of the idea, but it should be clear that this feature isn't moving forward and the exploration is not a production-ready polyfill that we recommend using.